### PR TITLE
SideBar

### LINKED ISSUE #7: side tool: design only (dummy data)

### PR DESCRIPTION
**Pull request by HassanChowdhry**
Screenshot 2023-07-06 at 9 41 25 PM

fixes #7

### CODE REVIEW

File: src/layout/SideBar.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+*/
+
+function SideBar() {
+  const theme = createTheme({
+    palette: {
+      primary: {
+        main: '#E1B15D', // brown
+      },
+      secondary: {
+        main: '#FFFFFF', // white
```
Comment (BilalIshtiaque) [inline src/layout/SideBar.tsx:20]: can you have 1 place for all our primary/secondary/tertiary colours ? and inherit/reference it...that way we dont have to make chagnes in multiple places

File: src/layout/SideBar.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { FilterList } from '@mui/icons-material';
+import FilterCheckbox from '../components/FilterCheckbox';
+
+// TODO: Needed to make fixed nav
+/* import { makeStyles } from '@material-ui/core/styles';
+ const useStyles = makeStyles((theme) => ({
+   toolbar: theme.mixins.toolbar,
+ }));
+*/
```
Comment (BilalIshtiaque) [inline src/layout/SideBar.tsx:11]: still need this ?

File: src/layout/SideBar.tsx (hunk 1 above)
Comment (HassanChowdhry) [reply src/layout/SideBar.tsx:20]: yea, just made a fix for it

File: src/layout/SideBar.tsx (hunk 2 above)
Comment (HassanChowdhry) [reply src/layout/SideBar.tsx:11]: nop, forgor to remove it

**Review by BilalIshtiaque — APPROVED** [review]

File: src/components/FilterCheckbox.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { FormControlLabel, Checkbox } from '@mui/material';
+
+interface Props {
+  label: string
+}
+
+function FilterCheckbox({ ...props }: Props) {
```
Comment (Ebrahim-Merchant) [inline src/components/FilterCheckbox.tsx:7]: why are you spreading the `props` here can't you just do e.g. with Component
```js
function FilterCheckbox(props: Props) {
 return <Component {...props} /> 
}
```

**Review by Ebrahim-Merchant — APPROVED** [review]

File: src/components/FilterCheckbox.tsx (hunk 3 above)
Comment (HassanChowdhry) [reply src/components/FilterCheckbox.tsx:7]: Did not notice lmao, ill commit the fix and merge.

**Pull request merged**